utils/install_tools.sh

```diff
@@ -60,37 +61,137 @@ install_cross ()
 		sudo dpkg -i advance-toolchain-at*cross-ppc*mcore* \
 			advance-toolchain-at*cross-ppc*extras* || return ${?}
 	else
-		sudo rpm -iv advance-toolchain-*common* || return ${?}
-		sudo rpm -iv advance-toolchain-*cross-ppc* || return ${?}
+		sudo rpm -${1}v advance-toolchain-*common* || return ${?}
+		sudo rpm -${1}v advance-toolchain-*cross-ppc* || return ${?}
 	fi
 }
 
 # Install AT packages.
 #
-# install <files_path> <at_version>
+# install <files_path> <at_version> <command>
 install ()
 {
 	pushd ${1}
 	shift
 
 	if [[ $(uname -m) == ppc* ]]; then
-		install_native ${@}
+		install_native ${1} ${2}
 	else
-		install_cross
+		install_cross ${2}
 	fi;
 	ret=${?}
 
 	popd
 	return ${ret}
 }
 
+# Install last AT version on PowerPC
+#
+# install_last_native <at_version>
+install_last_native ()
+{
+	local version=${1}
+
+	if [[ ${debian} == "yes" ]]; then
+		sudo apt-get install -y \
+		   advance-toolchain-at${version}-runtime \
+		   advance-toolchain-at${version}-runtime-dbg \
+		   advance-toolchain-at${version}-devel* \
+		   advance-toolchain-at${version}-mcore-libs* \
+		   advance-toolchain-at${version}-perf* \
```

Comment:
Why do you specify `-dbg` for runtime, but you using `*` for the rest of the packages?

Reply:
Because If we use * with the runtime we will get the compat packages too, they will conflict with the main runtime.